### PR TITLE
tcl/board: Add Spacemit MUSE Pi config

### PR DESCRIPTION
**Pull request by lz-bro**

### CODE REVIEW

File: tcl/board/spacemit-musepi.cfg [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+adapter driver jlink
+transport select jtag
+adapter speed 2000
+
+bindto 0.0.0.0
```
Comment (aap-sc) [inline tcl/board/spacemit-musepi.cfg:7]: to be fair , this does not belong to board/target configuration. 

```
adapter ...
transport ...
adapter ...
```

These are parameters of debug adapter and should be specified separately.

```
bindto ...
```
Is also a user preference.

**Comment by aap-sc** [comment]
@lz-bro I would also recommend to create a MR to upstream OpenOCD . I would prefer to merge back changes like these from upstream

**Comment by lz-bro** [comment]
> @lz-bro I would also recommend to create a MR to upstream OpenOCD . I would prefer to merge back changes like these from upstream

Thank you, I will create a MR to upstream OpenOCD.

**Comment by JanMatCodasip** [comment]
@lz-bro I can see that the patch to openocd.org has already been created: https://review.openocd.org/c/openocd/+/8361

For that reason, please consider closing this PR.

Thank you.

**Pull request closed without merging**